test/ctia/entity/entities_test.clj

```diff
@@ -1,23 +1,32 @@
 (ns ctia.entity.entities-test
   (:require [ctia.entity.entities :as sut]
-            [clojure.test :as t :refer [deftest is]]
+            [ctia.schemas.core :refer [MaybeDelayedRealizeFn->RealizeFn]]
+            [ctia.test-helpers.core :as test-helpers]
+            [ctia.http.server :refer [realize-fn-global-services]]
+            [clojure.test :as t :refer [deftest is use-fixtures join-fixtures]]
             [clojure.spec.alpha :refer [gen]]
             [clojure.spec.gen.alpha :refer [generate]]))
 
+(use-fixtures :each (join-fixtures [test-helpers/fixture-properties:clean
+                                    test-helpers/fixture-ctia-fast]))
+
 (defn gen-sample-entity
   [{:keys [new-spec]}]
   (if new-spec
     (generate (gen new-spec))
     {}))
 
 (deftest entity-realize-fn-test
-  (let [properties [:id :type :owner :groups :schema_version
+  (let [realize-fn-services (realize-fn-global-services)
+        properties [:id :type :owner :groups :schema_version
                     :created :modified :timestamp :tlp]
         ;; properties to dissoc to get a valid entity when
         ;; using the spec generator
         to-dissoc [:disposition]]
     (doseq [[_ {:keys [realize-fn] :as entity}] sut/entities]
-      (when realize-fn
+      (when-some [realize-fn (some-> realize-fn 
```

Comment:
Any way to strengthen this section so that we can ensure that tests are being run at all?

e.g. the `{:keys [realize-fn] :as entity}` destructuring may become stale at some point and the `when-some` would never pass

Reply:
Hmm I can't think of any way of strengthening this. `:realize-fn` an optional entry of an `ctia.schemas.core/Entity`, which `sut/entities` is asserted to have `Entity` keys in the `src`.

Reply:
Yeah it seems a bit hard tbh

Maybe sth like...?

```clj
(let [corpus (filter has-realize-fn? sut/entities)]
  (assert (seq corpus))
  (doseq [i corpus]
    ))
```

Reply:
Ah yes, I like that better thanks!

Reply:
Done. I also collapsed the nested `doseq`s.